src/app/work-item/work-item.service.ts

```diff
@@ -49,8 +49,10 @@ export class WorkItemService {
       return this.http
       .get(this.workItemTypeUrl)
       .toPromise()
-      .then((response) => {
-        this.workItemTypes = process.env.ENV != 'inmemory' ? response.json() as WorkItemType[] : response.json().data as WorkItemType[];
+      // .then((response) => {
+       // this.workItemTypes = process.env.ENV != 'inmemory' ? response.json() as WorkItemType[] : response.json().data as WorkItemType[];
+         .then(response => {
+        this.workItemTypes = response.json().data as WorkItemType[];
```

Comment:
And it doesn't :P 
<img width="1338" alt="screen shot 2016-12-03 at 9 18 45 am" src="https://cloud.githubusercontent.com/assets/2410474/20856625/94fcdb44-b939-11e6-9e68-73895c67ca46.png">